src/main/java/com/aws/greengrass/deployment/model/Deployment.java

```diff
@@ -83,6 +86,17 @@ public Deployment(DeploymentDocument deploymentDetails, DeploymentType deploymen
         this.deploymentStage = deploymentStage;
     }
 
+    // Get the deployment id created by GG cloud from deployment doc
```

Comment:
nit - could say something like -> `Get the deployment id set by GG cloud from deployment doc, this is different from the job id for job deployments`

Reply:
will add